content/en/agent/kubernetes/operator_configuration.md

```diff
@@ -28,550 +28,550 @@ spec:
+: If specified, indicates the pod's priority. `system-node-critical` and `system-cluster-critical` are two special keywords that indicate the highest priorities with the former being the highest priority. Any other name must be defined by creating a `PriorityClass` object with that name. If not specified, the pod priority will be default or zero if there is no default.
 
 `clusterAgent.rbac.create`
-: Used to configure RBAC resources creation.        
+: Used to configure RBAC resources creation.
 
 `clusterAgent.rbac.serviceAccountName`
-: Used to set up the service account name to use. Ignored if the field `Create` is true.    
+: Used to set up the service account name to use. Ignored if the field `Create` is true.
 
 `clusterAgent.replicas`
-: Number of the Cluster Agent replicas.             
+: Number of the Cluster Agent replicas.
 
 `clusterAgent.tolerations`
-: If specified, the Cluster Agent pod's tolerations.        
+: If specified, the Cluster Agent pod's tolerations.
 
 `clusterChecksRunner.additionalAnnotations`
-: Provide annotations that will be added to the cluster checks runner Pods.    
+: Provide annotations that will be added to the cluster checks runner Pods.
 
 `clusterChecksRunner.additionalLabels`
-: Provide labels that will be added to the cluster checks runner Pods. 
+: Provide labels that will be added to the cluster checks runner Pods.
 
 `clusterChecksRunner.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution`
-: The scheduler will prefer to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights, i.e. for each node that meets all of the scheduling requirements (resource request, `requiredDuringScheduling` affinity expressions, etc.), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding matchExpressions; the node(s) with the highest sum are the most preferred.     
+: The scheduler will prefer to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights, i.e. for each node that meets all of the scheduling requirements (resource request, `requiredDuringScheduling` affinity expressions, etc.), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding matchExpressions; the node(s) with the highest sum are the most preferred.
```

Comment:
```suggestion
: The scheduler prefers to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights. For each node that meets all of the scheduling requirements (resource request and `requiredDuringScheduling` affinity expressions), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding matchExpressions; the node(s) with the highest sum are the most preferred.
```